src/HSE/All.hs

```diff
@@ -171,8 +172,15 @@ parseModuleEx flags file str = timedIO "Parse" file $ do
                     pure $ Right (ModuleEx a' anns)
                 -- Parse error if GHC parsing fails (see
                 -- https://github.com/ndmitchell/hlint/issues/645).
-                GHC.PFailed _ loc err ->
-                    ghcFailOpParseModuleEx ppstr file str (loc, err)
+                GHC.PFailed s -> do
+                  -- TODO (SF, 2020-03-28): Quick patch here. After the
+                  -- migration to 8.10, this needs looking at more
+                  -- closely.
```

Comment:
I concur. Did you see my patch earlier today to the PFailed branch?

Reply:
No, I'll have a look

Reply:
Fixed this back to the old-style.